e2e/utils.ts

```diff
@@ -65,6 +65,11 @@ export function copyMissingPackages(): void {
     `rm -rf tmp/${projectName}/node_modules/@angular-devkit/core/node_modules`
   );
 
+  execSync(`rm tmp/${projectName}/node_modules/.bin/semver`);
```

Comment:
Why is this workaround required?

Reply:
`npm run e2e ngrx` was failing with *semver* errors. @FrozenPandaz  recommended this as the resolution.